Public/bmi-calculator.html

```diff
@@ -1,39 +1,46 @@
 <!DOCTYPE html>
-<html>
+<html lang="en">
 <head>
-   <title>BMI Calculator</title>
-   <link rel="stylesheet" type="text/css" href="../assets/css/bmi-calculator.css">
-
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>BMI Calculator</title>
+    <!--CSS Stylesheet-->
+    <link rel="stylesheet" href="./assets/CSS/bmi-calculator.css">
```

Comment:
use ../assts/css/
as your css isn't working now

Reply:
Okay @NOiR-07 I will resolve the issue and again make the pull request.

Reply:
issue #270 
I have resolved the issue and I made the changes in file.